doc/how_to_build_linux.md

```diff
@@ -1,5 +1,9 @@
 # Setting up the development environment for GNU/Linux
 
+## New: The Easy Way
+
+You can use the [local-build scripts](https://github.com/tahoma2d/tahoma2d/tree/master/ci-scripts/linux/local-build), which even eliminate the need to manually download this repository. These scripts automate the steps outlined below, making the process much easier.
+
 ## Required software
 
 Building Tahoma2D from source requires the following dependencies:
```

Comment:
This document suggests you'll need to get `blas` and `cblas` from AUR, but the new scripts added in this pull request do not do that. Can we make the documentation match the scripts?